src/components/elements/inputs/dropdown.js

```diff
@@ -2,14 +2,10 @@ import React from 'react'
 import Select from 'react-select'
 import Input from './input'
 
-export default class DropdownList extends Input {
+export default class Dropdown extends Input {
```

Comment:
A DropdownList sounds like a list of dropdowns to me. It's just a single dropdown.